api/src/main/java/net/kyori/adventure/audience/Audience.java

```diff
@@ -133,6 +135,34 @@
     return Audiences.COLLECTOR;
   }
 
+  /**
+   * Filters this audience.
+   *
+   * <p>The returned {@code Audience} may be the same, or a completely different one.</p>
+   *
+   * @param filter the filter
+   * @since 4.8.0
+   */
+  default @NonNull Audience filter(final @NonNull Predicate<? super Audience> filter) {
```

Comment:
Would it be better to call this `filterAudience`?

Reply:
`filterAudience`, to be consistent with above